api/capi/include/nnstreamer.h

```diff
@@ -38,6 +38,22 @@ extern "C" {
  * @{
  */
 
+/**
+ * @brief The virtual name to set the video source of camcorder in Tizen.
+ * @details If an application needs to access the camcorder to construct the pipeline, set the virtual name as a video source element.
+ *          Note that you have to add 'http://tizen.org/privilege/camera' into the manifest of your application.
+ * @since_tizen 5.5
```

Comment:
Is @since-tizen used in Tizen platform officially?
Or do you use theDoxygen tag as a custom tag instead @note or @todo?

Reply:
It is used for Tizen API.